libs/common-hooks/src/styled.d.ts

```diff
@@ -0,0 +1,11 @@
+import 'styled-components/macro'
+import { Theme } from '@cowprotocol/types'
+
+declare module 'styled-components/macro' {
+  export interface DefaultTheme extends Theme {}
+}
+
+// Re-export for the base package
+declare module 'styled-components' {
```

Comment:
This file should not be in common-hooks

Reply:
Yep, you're right. Removed it since these styled-component types are already properly defined in libs/types.